server/athenian/api/preloading/cache.py

```diff
@@ -366,6 +373,20 @@ async def preload(self, **dbs: databases.Database) -> None:
         await gather(*tasks)
         self._log.info("Finished preloading DB tables in memory")
 
+        self._log.debug("Scheduling refresh for preloaded DB tables in memory")
+        for db_name in get_memory_cache_options().keys():
```

Comment:
This will work, too - Go style:
```suggestion
        for db_name in get_memory_cache_options():
```